### DISCUSSION TITLE
Checkout Page Showing as Donations

### DISCUSSION THREAD

**Discussion by punafire** [jonas-hoebenreich/wc-donation-platform#38] [Q&A] [answered]
I sell products as well as accept donations on my woocommerce site. I added this plugin to accept the donations, but when someone places an order for a product (t-shirt, hat, etc.), the check out page shows as a donation, and when the order is placed, it's being logged as a donation, not an item purchase. Is there a way for the donation check out to only be when a someone selects to donate and when someone purchases an item it will show as a normal check out page?

**Comment by jonas-hoebenreich** [comment, marked as answer]
Hi punafire,

thanks for reaching out.
Donation Platform for WooCommerce turns WooCommerce into a donation system, parallel use as a normal store is not recommended, as some wording has been changed. In fact, it does not change the core functionality of WooCommerce, so you can continue to sell products normally. However, the wording in the user frontend has been adjusted. For example, the order summary is now called “Your Donation” instead of “Your Order”, the title of the order history and the texts of the emails have also been adjusted accordingly.
If this is not a problem for you, you can continue to sell normal products. Otherwise, my recommendation would be that you use a WordPress multisite or a separate WordPress installation for the donation page. Or you can use the free Loco Translate plugin to customize the texts.
see https://wcdp.jonh.eu/documentation/getting-started/faq/

Best
Jonas
